### PR TITLE
export in dec_time or std_time in export_pdf2

### PR DESCRIPTION
**Pull request by claudegel**
Changes proposed in this pull request:
- Add the possibility to export in standard time or decimal time in the pdf2 export extention

Reason for this pull request:
- This PR replace two old PR to rebase them to master, #544 and #543

### CODE REVIEW

**Comment by claudegel** [comment]
Once you merge the function change for gmdate in fix_export_summary maybe you can check this one to add the export in std_time for the pdf2 table format. It also allow for more info added in the table report by selecting (on/off) the column title in the export section.
This way we will have the possibility to have std_time export in both format, table or list.

File: extensions/ki_export/export_pdf2.php [hunk 1]
```diff
@@ -102,7 +102,16 @@ public function printRows($data, $widths) {
       else {
         $this->printTimeRow($widths, $row);
         $this->moneySum += $row['wage'];
-        $this->timeSum += $row['decimalDuration'] == -1 ? 0 : $row['decimalDuration'];
+        if ($_REQUEST['time_type']=="dec_time"){
```
Comment (kevinpapst) [inline extensions/ki_export/export_pdf2.php:105]: Can you please check for the code style in all your changes?!
There are a lot of whitespaces missing between operators, e.g.  
`_REQUEST['time_type']=="dec_time`
vs
`$_REQUEST['time_type'] == "dec_time`

**Comment by kevinpapst** [comment]
Hey, can you please sent the PR again, but against the "develop" branch.
Only hotfixes for critical bugs go to master.

**Comment by claudegel** [comment]
OK I'll redo it against develop and send new PR

**Pull request closed without merging**